packages/nextjs/app/builders/0x6C0FB309933931d111634921287aF902667032C4/page.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+
+import SwitchTheme from "./SwitchTheme";
+import { useTheme } from "next-themes";
+
+const NewPage = () => {
+  const { resolvedTheme } = useTheme();
+
+  return (
+    <div className="flex justify-center items-center h-screen bg-gray-100 dark:bg-gray-900 transition-colors duration-500">
+      <div
+        className="w-[600px] p-6 rounded-lg shadow-lg transition-all duration-500"
+        style={{
+          background:
+            resolvedTheme === "dark"
+              ? "linear-gradient(135deg, #8B0000, #000000)"
+              : "linear-gradient(135deg, #FF6347, #FFA500)",
+          color: resolvedTheme === "dark" ? "#ffffff" : "#000000",
+        }}
```

Comment:
Tailwind has built-in support for handling different themes using CSS:
https://v3.tailwindcss.com/docs/dark-mode.

It also provides utilities for linear gradients.

By avoiding the `useTheme` hook, you can remove `use client`, which improves performance by allowing your page to be pre-rendered at build time and served faster to users.